Source/Particles/Deposition/ChargeDeposition.H

```diff
@@ -13,10 +13,10 @@
 #include "Particles/Pusher/GetAndSetPosition.H"
 #include "Particles/ShapeFactors.H"
 #include "Utils/WarpXAlgorithmSelection.H"
-#include "Utils/WarpXProfilerWrapper.H"
 #ifdef WARPX_DIM_RZ
 #   include "Utils/WarpX_Complex.H"
 #endif
+#include "WarpX.H"
```

Comment:
The file uses `WarpX::shared_tilesize`.

Reply:
That's curious that the above change caused this to break - was it effectively including `WarpX.H` through the profile wrapper?

Reply:
Exactly. The profiler include masked the missing direct include.